### DISCUSSION TITLE
Drink selection tiles

### DISCUSSION THREAD

**Discussion by chellea** [nebhead/PiTender#16] [Ideas]
So, full disclosure, I just began working on this tonight...

I installed the code and have been looking around, and this looks great!!!  The only thing that really comes to mind is the drink menu.  How about a quick selection page with tiles for quick drink selection rather that scrolling through each drink?

Just a thought.

Thank you.

**Comment by nebhead** [comment]
Great thought, and would probably be easy enough to implement.  However, I may not get to this for a while as I have another project that is consuming more time.
